lib/devices/core.js

```diff
@@ -14,7 +14,7 @@ module.exports = function core(device) {
    * @return {void}
    */
   device.ping = function(callback) {
-    return command(commands.ping, null, callback);
+    command(commands.ping, null, callback);
```

Comment:
I'm unsure if we want them there or not, but I'd added the `return`s to device commands to make it easier if we wanted to add a Promise interface in the future or something. I'll leave it to you whether or not they should be there for now.